editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3766,7 +3766,7 @@ bool CanvasItemEditorViewport::_cyclical_dependency_exists(const String &p_targe
 
 void CanvasItemEditorViewport::_create_nodes(Node *parent, Node *child, String &path, const Point2 &p_point) {
 	child->set_name(path.get_file().get_basename());
-	Ref<Texture> texture = Object::cast_to<Texture>(Ref<Texture>(ResourceCache::get(path)).ptr());
+	Ref<Texture> texture = Ref<Texture>(Object::cast_to<Texture>(ResourceCache::get(path)));
```

Comment:
Thanks for catching this! I'll go through the diff again and make sure I didn't make this mistake elsewhere (I apparently saw Ref and decided I needed to dereference, whoops)

Reply:
Anytime! You did fix a lot of issues in your pull request.

For what it's worth, I did a quick `egrep` to catch if there was another line like that:

```
egrep -R "Object::cast_to<.*>\(.*Ref"
```

but nothing showed up. I think the rest is good.